megamek/docs/history.txt

```diff
@@ -16,11 +16,6 @@ v0.45.5-SNAPSHOT
 + Data: New Boards from new Beginner Box set (thanks ask)
 + Data: New units from TP:Tokasha, OTP:Fronc Reaches and Vega 3039
 + Issue #1090: VGL Unable to Fire - Out of Arc
-+ Issue #1375: Heat damage not correctly applied to all unit types
```

Comment:
I merged in the latest changes to Master and I think doing so resolved this problem. Let me know if it looks ok now.
As a general rule, I don't make changes to anything but the code (no images, text files, etc), so when something like this sneaks in it's coming from whatever state the main branch was in when I merged it in. 
That said, I'm not sure how best to prevent that - suggestions?

Reply:
I try to regularly do a "git pull upstream master" on my workspace so that my files are up to date with the upstream repository.

Reply:
I try to regularly do a "git pull upstream master" so my local workspace stays up to date with the upstream repository.

Reply:
If your pull strategy is `merge` that will still bring in that sort of change (as a merge commit).

If your pull strategy is `rebase` your work will appear to have been done after the changes on upstream/master. However, pull with rebase is advanced and probably best not enabled unless you're comfortable with a rebase workflow.